### PR TITLE
add unexported vars storing pango/cairo version literals

### PR DESCRIPTION
**Pull request by bjarthur**
i would add similar for gobject and pangocairo but there doesn't seem to be a way to query their version at runtime.

### CODE REVIEW

**Comment by lobingera** [comment]
a) Glib has https://developer.gnome.org/glib/2.50/glib-Version-Information.html#glib-check-version and Gobject follows, as it's a part of Glib
b) the same applies for pangocairo which is an integral part of pango

**Comment by bjarthur** [comment]
thanks.  but calling `glib_major_version` is crashing the windows appveyor builds.  works fine on linux and mac.  should i add `if !is_windows()` logic, or is there a way to get the version on that platform?

**Comment by lobingera** [comment]
> but calling glib_major_version is crashing the windows appveyor builds

afaics in the docs, that's not a function you call, you just access the symbol to get the data, maybe via cglobal?

**Comment by bjarthur** [comment]
i'm using cglobal already to get glib_major_version.  works great on linux and mac.  can anyone with a windows box help debug?

**Comment by bjarthur** [comment]
i added logic to not set `libpangocairo_version` and `libgobject_version` on windows.  `libcairo_version` and `libpango_version` are set on all platforms.

since tests now pass, i will merge in a couple days unless someone volunteers to debug the windows inconsistencies.

**Comment by lobingera** [comment]
@bjarthur What are you actually doing with these version numbers?

File: src/Cairo.jl [hunk 1]
```diff
@@ -13,6 +13,19 @@ using Colors
 importall Graphics
 import Base: copy
 
+libcairo_version = VersionNumber(unsafe_string(
+      ccall((:cairo_version_string,Cairo._jl_libcairo),Cstring,()) ))
+libpango_version = VersionNumber(unsafe_string(
+      ccall((:pango_version_string,Cairo._jl_libpango),Cstring,()) ))
+if !is_windows()
+  libpangocairo_version = VersionNumber(unsafe_string(
```
Comment (tkelman) [inline src/Cairo.jl]: most of the rest of the package uses 4 space indent

**Comment by bjarthur** [comment]
they're useful for debugging.  i recently had cairo work on one machine and not another so coded up how to query the version info to see if that was the cause.  figured i'd share with everyone else to make it easier in future.

note also the use of version info in `test/test_speed.jl`.

plus other packages, `Gtk.jl` for example, have similar variables.  i think it's generally useful.

**Pull request merged**